e2e/cypress/integration/enterprise/system_console/system_scheme_spec.js

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import users from '../../../fixtures/users.json';
+
+import {deleteExistingTeamOverrideSchemes} from './team_scheme_spec.js';
```

Comment:
When we import a specific function from another test, Cypress runs the entire imported test like this:
![Screenshot 2020-03-24 at 4 32 41 PM](https://user-images.githubusercontent.com/1429138/77418692-41fd0880-6ded-11ea-9af7-df879490f1f4.png)

So it is suggested to have the function local in multiple tests even if its redundant.

Reply:
done